tests/test_enum.py

```diff
@@ -136,6 +136,19 @@ def test08_enum_comparisons():
     assert not (t.Enum.B == t.SEnum.B) and not (t.SEnum.B == t.Enum.B)
     assert t.Enum.B != t.SEnum.C and t.SEnum.C != t.Enum.B
 
+def test06_enum_flag():
+    assert (t.Flag(1) | t.Flag(2)).value == 3
```

Comment:
Tests for string conversion are missing.

Reply:
Added above in [599b191](https://github.com/wjakob/nanobind/pull/688/commits/599b191e84be663ecf4ed8598344243ebac176ac).